docs/examples/JSONExample/src/JSONExample.ls

```diff
@@ -90,11 +123,51 @@ package
             sprite.x = 240;
             sprite.y = 120;
             stage.addChild(sprite);
+            
+            
+            var json = new JSON();
+            
+            json.initObject();
+            
+            
+            // Circular reference tests
```

Comment:
Wait a second, shouldn't the test / build server automatically set the version to dev while testing? So I'm not sure why the test would fail? Seems green too.